app/helpers/curation_concern_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module CurationConcernHelper
+  # https://cits.artic.edu/redmine/issues/2183
```

Comment:
Nice work on adding the ticket number. Not bad if we every need to dig up correspondence.